cmd/build/command_test.go

```diff
@@ -37,26 +40,194 @@ func getManifestWithError(_ string) (*model.Manifest, error) {
 	return nil, assert.AnError
 }
 
-func getFakeManifest(_ string) (*model.Manifest, error) {
-	return fakeManifest, nil
+func getManifestWithInvalidManifestError(_ string) (*model.Manifest, error) {
+	return nil, oktetoErrors.ErrInvalidManifest
+}
+
+func getFakeManifestV1(_ string) (*model.Manifest, error) {
+	manifestV1 := *fakeManifestV2
+	manifestV1.IsV2 = false
+	return &manifestV1, nil
+}
+
+func getFakeManifestV2(_ string) (*model.Manifest, error) {
+	return fakeManifestV2, nil
+}
+
+func TestIsBuildV2(t *testing.T) {
+	tests := []struct {
+		name           string
+		manifest       *model.Manifest
+		expectedAnswer bool
+	}{
+		{
+			name: "manifest v1 is build v1",
+			manifest: &model.Manifest{
+				IsV2: false,
+			},
+			expectedAnswer: false,
+		},
+		{
+			name: "manifest v2 with no build section is build v1",
+			manifest: &model.Manifest{
+				IsV2:  true,
+				Build: model.ManifestBuild{},
+			},
+			expectedAnswer: false,
+		},
+		{
+			name: "manifest v1 with build section is build v1",
+			manifest: &model.Manifest{
+				IsV2: false,
+				Build: model.ManifestBuild{
+					"test-1": &model.BuildInfo{
+						Image: "test/test-1",
+					},
+					"test-2": &model.BuildInfo{
+						Image: "test/test-2",
+					},
+				},
+			},
+			expectedAnswer: false,
+		},
+		{
+			name: "manifest v1 with build section is build v1",
+			manifest: &model.Manifest{
+				IsV2: false,
+				Build: model.ManifestBuild{
+					"test-1": &model.BuildInfo{
+						Image: "test/test-1",
+					},
+					"test-2": &model.BuildInfo{
+						Image: "test/test-2",
+					},
+				},
+			},
+			expectedAnswer: false,
+		},
+		{
+			name: "manifest v2 with build section is build v2",
+			manifest: &model.Manifest{
+				IsV2: true,
+				Build: model.ManifestBuild{
+					"test-1": &model.BuildInfo{
+						Image: "test/test-1",
+					},
+					"test-2": &model.BuildInfo{
+						Image: "test/test-2",
+					},
+				},
+			},
+			expectedAnswer: true,
+		},
+	}
+	for _, tt := range tests {
```

Comment:
I think you should do something like this, it will be a race condition if it's not done
```suggestion
	for _, tt := range tests {
	  tt := tt
```